openshift/ci-operator/knative-images/cli-artifacts/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# This is not generated Dockerfile, yet!
+ARG GO_RUNTIME=registry.access.redhat.com/ubi8/ubi-minimal
+ARG CLI_ARTIFACTS=registry.redhat.io/openshift-serverless-1/kn-cli-artifacts-rhel8:1.15
```

Comment:
We probably have 1.16 available now too (this was a leftover, as no 1.17 was available when the file was copied IIRC :D )

Reply:
Ah great I was wondering about this, will bump